packages/backend/src/App.ts

```diff
@@ -181,7 +198,7 @@ export default class App {
 
         const expressApp = await this.initExpress();
         this.initSentry(expressApp);
-        this.initSlack();
+        await this.initSlack();
```

Comment:
do we need to await for this?

Reply:
There are 3 options:
`void` - the promise doesn't block the thread but can fail silently and trigger an "unhandled promise error"
`await` - the promise blocks the thread but the error is handled
`.catch(...)` - the promise doesn't block the thread and the error is handled.

I tend to await all promises and find `void` to be the worst option of all.